spec/serializers/service_provider_serializer_spec.rb

```diff
@@ -35,12 +35,24 @@
           eq(service_provider.email_nameid_format_allowed)
         expect(as_json[:signed_response_message_requested]).to \
           eq(service_provider.signed_response_message_requested)
-        expect(as_json[:allow_prompt_login]).to eq(true)
+        expect(as_json[:allow_prompt_login]).to be true
```

Comment:
Curious about the difference between `be`  and `eq` here

Reply:
`be` is usually used with more fluid matchers, ex you can say `be >= 1` and it would assert `>=` and there's also the `be_` set of magical prefixes like `be_truthy` and `be_anything` (which would try to call `obj.anything?`)

Reply:
that makes sense thanks @zachmargolis  @Sgtpluck  !